### DISCUSSION TITLE
Multiple actions and/or links in a alert banner

### DISCUSSION THREAD

**Discussion by cdpestitschek** [twilio-labs/paste#2250] [Q&A] [answered]
I would like to have guidance recommendations for in the context of having multiple links in the alert banner.

Since they are multiple links, and also clarifying whether in the existence of more than one link we will have a distinction between primary and secondary action. In the [documentation](https://paste.twilio.design/components/alert/#error) i just found the case of the nav bar supporting a link.

### User statement

> As a Flex customer I want to be notified when Flex initialises in degraded mode
> So I can expect unexpected Flex behaviours

### Image examples

**Before**
Applying the Paste link format, it seems that it is just 1 action, but there are two different actions
<img width="1080" alt="Screenshot 2022-02-17 at 3 16 10 pm" src="https://user-images.githubusercontent.com/100429369/156010268-22ae310a-a704-4fe8-82e0-2da7e0ff3770.png">

**Now**
Temporarily it is being presented this way: highlighting the distinction of the two actions
<img width="1777" alt="Screenshot 2022-02-28 at 2 18 26 pm" src="https://user-images.githubusercontent.com/100429369/156010299-059e62fa-a5cd-4d5f-8c92-84a21b62461c.png">

**Stage of design:** Implemented
**Research:** Include links to any findings
**Links to mockups or prototype:** (please allow commenting if possible)
**Git:** https://issues.corp.twilio.com/browse/FLEXAPP-106

**Comment by nkrantz** [comment, marked as answer]
Hey @cdpestitschek - the way you're currently displaying the two links in the Alert with a divider between them works, but if you're worried about the way it looks (not enough visual distinction between the two), you can use a [Small Button](https://paste.twilio.design/components/button/#small-button) for the "Refresh" action. Both options are good, and feel free to play around with other ways to display these 2 actions in the Alert. The contents of an Alert are composable, so you can use the components you need.

Just a note - in your first screenshot, it looks like the Alert is covering up the very top part of the UI ("Admin dashboard" is cut off on the top). Remember that Alerts should never cover any elements on the page - see [the Alert docs](https://paste.twilio.design/components/alert/#when-to-use-an-alert) for more guidance on this.

Hope this helps, and please let us know if you have more questions!

**Reply by shleewhite** [reply]
Hi @cdpestitschek Do you have what you need or would you like more assistance with this?

**Reply by cdpestitschek** [reply]
Not for now @shleewhite. @nkrantz thank you so much for the feedback && clarification.
